Source/WebInspectorUI/UserInterface/Controllers/ConsoleManager.js

```diff
@@ -162,6 +164,8 @@ WI.ConsoleManager = class ConsoleManager extends WI.Object
             let issue = new WI.IssueMessage(message);
             this._issues.push(issue);
 
+            this._collectFailedSourceMapConsoleMessage(message);
```

Comment:
please move this to the end of the `if` since it has nothing to do with the `issue` that's created above and used below

Reply:
Addressed in e2c3874b5b35844fbcd028d68867b71662ae988e.